sklearn/feature_selection/_univariate_selection.py

```diff
@@ -51,16 +51,16 @@ def f_oneway(*args):
 
     Parameters
     ----------
-    *args : array-like, sparse matrices
+    *args : {array-like, sparse matrix}
         sample1, sample2... The sample measurements should be given as
         arguments.
 
     Returns
     -------
-    F-value : float
+    f_statistic : float
         The computed F-value of the test.
-    p-value : float
-        The associated p-value from the F-distribution.
+    p_value : float
+        The associated P-value from the F-distribution.
```

Comment:
I think the original notation is more common:

```suggestion
        The associated p-value from the F-distribution.
```

https://en.wikipedia.org/wiki/P-value#cite_note-2